101-custom-rp-with-function/azuredeploy.json

```diff
@@ -27,19 +21,21 @@
             "type": "string",
             "defaultValue": "[uniqueString(resourceGroup().id)]",
             "metadata": {
-                "description": "Name of the storage account for storing users in the function app"
+                "description": "Name of the storage account for storing custom resources in the function app."
             }
         },
-        "userName": {
+        "azureCustomResourceName": {
             "type": "string",
+            "defaultValue": "[uniqueString(newGuid())]",
```

Comment:
this doesn't need to be globally unique - why not just use the value we use in the readme to provide continuity?